src/Types/index.tsx

```diff
@@ -3,6 +3,8 @@ export interface StandardProps {
   className?: string
   /** ID for Integration Tests */
   testID: string
```

Comment:
should this be optional?

Reply:
It has a default value on all the components